Jenkinsfile

```diff
@@ -1,5 +1,10 @@
 pipeline {
     agent none
+    environment {
+        COMPONENT_NAME = 'DevHub web app'
+        COMPONENT_HOME = '/'
+        BUILD_TRIGGER_EXCLUDES = "^.jenkins/\\|^matomo/"
```

Comment:
can this use an INCLUDES regex over the excludes? is it because it is a root project jenkinsfile?

Reply:
Yes - in this case, it requires an excludes because there is/was code in the root that related to DevHub.  Once the folder restructure is in place it can just be an 'include'.  

We can definitely come up with something better for this, but IMO it ultimately belongs outside the Jenkinsfile and integrated/generated by bcdk.